zboss/CHANGELOG.rst

```diff
@@ -12,6 +12,12 @@ All notable changes to this project in |NCS| are documented in this file.
 Master branch
 *************
 
+Added
+=====
+
+* Second set of ZBOSS libraries, dedicated for the NCP development.
+  Those libraries are based on the 3.4.0.769 ZBOSS build.
+
 Changes
 =======
 
```

Comment:
Don't we also update the ZBOSS production libraries to 3.3.0.7?

Reply:
No, this PR touches only one set of libraries.